TESTING.md

```diff
@@ -302,3 +322,40 @@ cargo make -e TESTSYS_TEST=migration test
 ```
 
 To see the state of the tests as they run use `cargo make watch-test`.
+
+### Custom Test Types
+
+Custom tests can be run with TestSys by calling `cargo make -e TESTSYS_TEST=<CUSTOM-TEST-NAME> test -f <PATH-TO-TEMPLATED-YAML>`.
+
+First, a test agent needs to be constructed.
+The `test-agent-cli` provides an interface for creating bash based testing agents.
+Checkout the [runbook](https://github.com/bottlerocket-os/bottlerocket-test-system/blob/v0.0.3/agent/test-agent-cli/design/RUNBOOK.md) for instructions on creating an agent.
```

Comment:
I think pointing to a specific version is brittle because people won't know to update it when we move to a newer version of testsys.